### ISSUE TITLE
`include_multiline_strings: false` breaks with string interpolation in multiline strings

### ISSUE THREAD

**Issue by robertlacroix** [realm/SwiftLint#4951]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### Describe the bug

#4785 added support for excluding multiline strings from `indentation_width` checks. It works great for purely string cases, but breaks with string interpolation in multiline strings.

##### Complete output when running SwiftLint, including the stack trace and command used

```bash
$ swiftlint lint test.swift --config indent.yml
Linting Swift files at paths test.swift
Linting 'test.swift' (1/1)
test.swift:4:1: warning: Indentation Width Violation: Code should be indented using one tab or 4 spaces (indentation_width)
Done linting! Found 1 violation, 0 serious in 1 file.
```

### Environment

* SwiftLint version (run `swiftlint version` to be sure)? 0.51.0
* Installation method used (Homebrew, CocoaPods, building from source, etc)? building from source

indent.yml:
```yml
opt_in_rules:
  - indentation_width
indentation_width:
  include_multiline_strings: false
```

test.swift:
```swift
func test {
    let multilineWithStringInterpolation = """
    \\{
      "schemaVersion" : \(expectedSchema)
    \\}
    """

    let multilineWithoutStringInterpolation = """
    \\{
      "schemaVersion" : 1
    \\}
    """
}
```

**Comment by mildm8nnered** [comment]
I'm taking a quick look at this today

**Comment by mildm8nnered** [comment]
ok - this was resolved by #4862 already, so this ticket can be closed as a dupe. The fix is in `main`, and will be in the next SwiftLint release.

**Comment by robertlacroix** [comment]
Great, thanks for checking!
